certbot-nginx/certbot_nginx/_internal/parser.py

```diff
@@ -107,11 +113,11 @@ def _build_addr_to_ssl(self):
                     addr_to_ssl[addr_tuple] = addr.ssl or addr_to_ssl[addr_tuple]
         return addr_to_ssl
 
-    def _get_raw_servers(self) -> Dict:
+    def _get_raw_servers(self) -> Dict[str, Union[List[Any], nginxparser.UnspacedList]]:
```

Comment:
Since it is imported, we may as well do this across the file?

```suggestion
    def _get_raw_servers(self) -> Dict[str, Union[List[Any], UnspacedList]]:
```